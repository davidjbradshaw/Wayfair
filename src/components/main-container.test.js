import MainContanier from './drawer-container'

test('renders MainContanier', () => {
  expect(<MainContanier />).toMatchSnapshot()
})
