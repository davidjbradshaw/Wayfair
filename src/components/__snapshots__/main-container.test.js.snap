// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MainContanier 1`] = `<DrawerContainer />`;
